etl_manager/etl.py

```diff
@@ -137,63 +134,65 @@ def etl_root_folder(self):
 
     @property
     def job_arguments(self):
-        metadata_argument = {"--metadata_base_path": self.s3_metadata_base_folder_inc_bucket}
+        metadata_argument = {
+            "--metadata_base_path": self.s3_metadata_base_folder_inc_bucket,
```

Comment:
☝️ think the comma may break when you try to run a glue job. Annoyingly Travis skips running some of the job runner stuff so maybe why this passed?

Reply:
My mistake apparently that's fine - all good from me